### DISCUSSION TITLE
Pin sniffing processes to the same CPU that your sniffing NIC is bound to

### DISCUSSION THREAD

**Discussion by sleepingbel** [Security-Onion-Solutions/securityonion#2235] [Unsupported Versions] [answered]
Hello,

SO 2.3.10 ISO
8 vCPU
16 vRAM
2 TB vDisk Space

I have zeek capture loss and want to pin the workers to the same cpu as the sniffing NIC, because by increasing only the workers the packet loss gets bigger. But I can not find the file node.cfg on the location as stated on the zeek documentation (/etc/node.cfg).
[https://docs.securityonion.net/en/2.3/zeek.html#performance](url)
[https://docs.zeek.org/en/current/configuration/#using-pf-ring](url)

I have searched in the zeek container and in the main etc directory of SO

Can you give the location of the node.cfg file?

Is it possible that this capture loss is caused by the mirror port on the switch and router?
Because the suricata and zeek packet loss gives 0%.

Regards
Bart

**Comment by dougburks** [comment, marked as answer]
If Zeek packet loss is 0% but you have Zeek capture loss, then that most likely indicates upstream loss at whatever device your sniffing interface is connected to.

**Comment by presianbg** [comment]
Hi,

How can I found which CPU the sniffing NIC is bound to ?
